lib/dor/models/governable.rb

```diff
@@ -27,7 +27,7 @@ def default_workflow_lane
     end
 
     def reset_to_apo_default
-      rightsMetadata.content = admin_policy_object.rightsMetadata.ng_xml
+      rightsMetadata.content = admin_policy_object.rightsMetadata.ng_xml.to_s
```

Comment:
There was a missing spec on this method and, when it was created/enabled, it failed.  The `.to_s` fixed it.